rcl/test/CMakeLists.txt

```diff
@@ -162,14 +183,16 @@ function(test_target_function)
     AMENT_DEPENDENCIES ${rmw_implementation}
   )
 
-  rcl_add_custom_gtest(test_node${target_suffix}
-    SRCS rcl/test_node.cpp
-    ENV ${rmw_implementation_env_var} ${memory_tools_ld_preload_env_var}
-    APPEND_LIBRARY_DIRS ${extra_lib_dirs}
-    LIBRARIES ${PROJECT_NAME} mimick osrf_testing_tools_cpp::memory_tools
-    AMENT_DEPENDENCIES ${rmw_implementation} "osrf_testing_tools_cpp"
-    TIMEOUT 240  # Large timeout to wait for fault injection tests
-  )
+  if(NOT "${DISTRIBUTION}" STREQUAL "\"centos\"")
```

Comment:
@clalancette nit: a small note explaining this would be nice.

Reply:
Good point, will add a note.

Reply:
Added a note in 111cb41